src/connectors/playirish.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+Connector.playerSelector = '.qt-musicplayer';
+
+Connector.artistTrackSelector = '#streamCurrentlyPlaying';
+
+Connector.getArtistTrack = () => {
+	let text = $(Connector.artistTrackSelector).text().replace('NOW PLAYING:', '');
+	return Util.splitArtistTrack(text);
+};
+
+Connector.playButtonSelector = '.sm2_playing';
```

Comment:
I'm sorry, I was wrong when I suggested to use this selector as play button selector. There should be `Connector.pauseButtonSelector` instead. Currently the extension detects playing state as paused.